site/content/en/docs/concepts/multikueue.md

```diff
@@ -82,5 +82,5 @@ Kueue delegates the job to the configured worker clusters without any additional
 
 ## What’s next? 
 - Learn how to [setup a MultiKueue environment](/docs/tasks/manage/setup_multikueue/)
-- Learn how to [submit JobSets](/docs/tasks/run/jobsets/#jobset-definition) to a running Kueue cluster.
-- Learn how to [submit batch/Jobs](/docs/tasks/run/jobs/#1-define-the-job) to a running Kueue cluster.
+- Learn how to [submit JobSets](/docs/tasks/run/multikueue/jobsets) to a running MultiKueue cluster.
+- Learn how to [submit batch/Jobs](/docs/tasks/run/multikueue/jobs) to a running MultiKueue cluster.
```

Comment:
```suggestion
- Learn how to [run jobs](/docs/tasks/run/multikueue) in MultiKueue environment.
```

We do not want to list all jobs since there are so many job types.